content/participants/ttjc@Harshita-Kanal/index.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Harshita-Kanal
+type: "participant"
+name: "Harshita Kanal"
+portfolio: "https://github.com/Harshita-Kanal"
+github-link: "https://github.com/Harshita-Kanal"
+linkedin-link: "https://www.linkedin.com/in/harshita-kanal-400563166/"
+looking-for: "Internship"
+description: "React | Node | Express | Javascript | Developer"
+---
+
+Hello, I am Harshita Kanal, I am passionate about building things that impact people's lives. That is one of the reasons why I entered this field. I enjoy connecting with people, taking challenges, and I am always open to new opportunities, ventures, or anything which can put my skills to test.
+
+### introduction
+
+I am a Computer Science and Engineering undergraduate at Viviekanand Education society's Institute of Techology. Currently, I am in my second year.I have participated in various hackathons, and coding competitions. I am a passionate and hardworking individual. I can assure you that the work assigned to me would be delivered to your due satisfaction. 
```

Comment:
* technology
* hard working

Reply:
I have made the changes, please check the latest commit.